content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -177,44 +177,60 @@ On the first usage of a GDrive [remote](/doc/command-reference/remote), for
 example when trying to `dvc push` tracked data for the first time, DVC will
 prompt you to visit a special Google authentication web page. There you'll need
 to sign into a Google account with the needed access to the GDrive
-[URL](#url-format) in question. The
-[auth process](https://developers.google.com/drive/api/v2/about-auth) will ask
-you to grant DVC the necessary permissions, and produce a verification code
-needed for DVC to complete the connection. On success, the necessary credentials
-will be saved in a Git-ignored file, located in
-`.dvc/tmp/gdrive-user-credentials.json` and they will be used automatically next
-time you run DVC.
-
-⚠️ In order to prevent unauthorized access to your Google Drive, **do not share
+[URL](#url-format) in question. The [auth process] will ask you to grant DVC the
+necessary permissions, and produce a verification code needed for DVC to
+complete the connection. On success, the necessary credentials will be cached
+globally ([see `gdrive_user_credentials_file`]) and used automatically next time
```

Comment:
let's put a proper example and see also ...

Reply:
Not exactly sure what you mean. Proper example = `~/Library/Caches/...`? If so please commit:

```suggestion
globally, for example in
`~/Library/Caches/pydrive2fs/{gdrive_client_id}/default.json` for Linux
([see `gdrive_user_credentials_file`]), and used automatically next time
```

Reply:
@jorgeorpinel That's incorrect for linux. It is more like "~/.cache/pydrife2fs/{gdrive_client_id}/default.json", as indicated by the table
<img width="529" alt="Screenshot 2022-07-29 at 20 53 42" src="https://user-images.githubusercontent.com/5367102/181817112-265cacb1-4cc9-4776-b621-04c6e33e2211.png">

Reply:
Right, I meant macOS. Changed. Committing...